apps/sms/views/shared/style/root.css

```diff
@@ -12,7 +12,7 @@ body {
   font-size: 1.6rem;
   line-height: 2.2rem;
   vertical-align: middle;
-  background-color: #FFF;
+  background: var(--header-background);
```

Comment:
I don't think so; I think the trick here is especially to use the same color than the header so that we see no flickering :)